### PR TITLE
CalLogs - Switch to Prod code

### PR DESCRIPTION
**Pull request by Shanefe**
**Issue:**
Moving key functions into the Dev code.  Switch script to using them. 

**Reason:**
LogType --> LogRowType
client --> ShortClientInfoString

**Validation:**
Testing against Test tenants.

### CODE REVIEW

File: Calendar/Get-CalendarDiagnosticObjectsSummary.ps1 [hunk 1]
```diff
@@ -162,14 +163,13 @@ if (-not ([string]::IsNullOrEmpty($Subject)) ) {
...
                     }
-                    Write-Host -ForegroundColor Cyan "`t Ignore the next [$($LogToExamine.count)] warnings..."
                     $logLeftCount = $LogToExamine.count
 
                     $ExceptionLogs = $LogToExamine | ForEach-Object {
                         $logLeftCount -= 1
                         Write-Verbose "Getting Exception Logs for [$($_.ItemId.ObjectId)]"
-                        Get-CalendarDiagnosticObjects -Identity $ID -ItemIds $_.ItemId.ObjectId -ShouldFetchRecurrenceExceptions $true -CustomPropertyNames $CustomPropertyNameList -ShouldBindToItem $true
-                        if ($logLeftCount % 20 -eq 0) {
+                        Get-CalendarDiagnosticObjects -Identity $ID -ItemIds $_.ItemId.ObjectId -ShouldFetchRecurrenceExceptions $true -CustomPropertyNames $CustomPropertyNameList -ShouldBindToItem $true 3>> $null
```
Comment (dpaulson45) [inline Calendar/Get-CalendarDiagnosticObjectsSummary.ps1:171]: What is this doing? Why are we adding `3>> $null` here?

Comment (dpaulson45) [reply Calendar/Get-CalendarDiagnosticObjectsSummary.ps1:171]: Okay, looks like we are just trying to get rid of the warning stream. If that is the case, we should then use `-WarningAction SilentlyContinue` to make it a little easier to read.

Comment (Shanefe) [reply Calendar/Get-CalendarDiagnosticObjectsSummary.ps1]: Sweet, that worked.  I thought I had tried that before.

**Review by dpaulson45 — APPROVED** [review]

**Pull request merged**